package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "pretest": "mkdir -p node_modules/ && ln -fs $(pwd) node_modules/",
     "eslint-check": "eslint-config-prettier .eslintrc.js",
-    "test": "npm run eslint-check && eslint . && mocha tests/"
+    "test": "npm run eslint-check && eslint . && mocha tests/**/*.js tests/"
```

Comment:
I thought `tests/**/*.js` will run all tests including ones in nested folder but it _only_ ran the nested tests. Therefore I had to specify `tests/` too. Open to suggestion